doc/manuals/admin/database_model.md

```diff
@@ -343,6 +343,7 @@ notifications. It is a number between 0 and 1800000. If defined to 0 or omitted,
 -   **covered**: a boolean field that specifies if all `attrs` have to be included in notifications (if value is true)
     or only the ones existing in the triggering entity (if value is false or field is omitted).
     More information in [covered subscription section in Orion API specification](../orion-api.md#covered-subscriptions).
+-   **notifyOnMetadataChange**: a boolean variable that defines to trigger notifications only for metadata changes (if the value is true) or If only the metadata changes, avoid trigger notifications (if the value is false).
```

Comment:
```suggestion
-   **notifyOnMetadataChange**: if `true` metadata is considered part of the value of the attribute regarding subscription triggering. If `false` metadata is not considered part of the value of the attribute regarding subscription triggering. Default value (if omitted) is `true`.
```

Reply:
Fixed in [3ba1ead](https://github.com/telefonicaid/fiware-orion/pull/4310/commits/3ba1ead3128937e80932ee1f3210bacc23f3d475)